### PR TITLE
Tmo50 initial commit

### PR DESCRIPTION
**Pull request by emiilsd**
Commiting initial "default" keymap for TMO50 50% mechanical keyboard

### CODE REVIEW

File: keyboards/tmo50/config.h [hunk 1]
```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2018 funderburker
```
Comment (mechmerlin) [inline keyboards/tmo50/config.h]:
```suggestion
Copyright 2019 funderburker
```

File: keyboards/tmo50/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/* Copyright 2018 funderburker
```
Comment (mechmerlin) [inline keyboards/tmo50/keymaps/default/keymap.c]:
```suggestion
/* Copyright 2019 funderburker
```

File: keyboards/tmo50/rules.mk [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Boot section
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (noroadsleft) [inline keyboards/tmo50/rules.mk]: The `BOOTLOADER` rule sets this option automatically.

You can comment out Line 45, or delete it entirely.

File: keyboards/tmo50/tmo50.h [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef TMO50_H
```
Comment (noroadsleft) [inline keyboards/tmo50/tmo50.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`?

It's simpler, and less prone to user error, so if you could make this switch?

```suggestion
#pragma once
```

File: keyboards/tmo50/tmo50.h [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+#define TMO50_H
```
Comment (noroadsleft) [inline keyboards/tmo50/tmo50.h]:
```suggestion
```

File: keyboards/tmo50/tmo50.h [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+	K300, K301, K302, K303,       K305,       K307,       K309, K310  \
+) \
+{ \
+	{ K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013 }, \
+	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113 }, \
+	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213 }, \
+	{ K300,  K301,  K302,  K303,  KC_NO, K305,  KC_NO, K307,  KC_NO, K309,  K310,  KC_NO, KC_NO, KC_NO }  \
+}
+
+#endif
```
Comment (noroadsleft) [inline keyboards/tmo50/tmo50.h]:
```suggestion
```

File: keyboards/tmo50/rules.mk (hunk 3 above)
Comment (emiilsd) [reply keyboards/tmo50/rules.mk]: Removed and commited.

File: keyboards/tmo50/tmo50.h (hunk 4 above)
Comment (emiilsd) [reply keyboards/tmo50/tmo50.h]: Changed include guard

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**